File: ws/lascano/u2/ws17Libraries/TaxesLibWithMaven/src/main/java/utils/Tax.java
package utils;

/**
 *
 * @author Edison Lascano, Object Masters, @ESPE
 */
public class Tax {
    public static float computeIva(float amount, float taxPercentage){
        float taxValue;
        taxValue = amount * taxPercentage/100;
        return taxValue;
    }
    
    public static float computeTotal(float amount, float taxPercentage){
        float totalValue;
        totalValue = amount + computeIva(amount, taxPercentage);
        return totalValue;
    }
}

File: hw/nacato/u2/hw09StoreApp/StoreApp.py/libraries/Tax.py
class Tax:
    
    @staticmethod
    def computeIva(amount, taxPercentage):
        taxValue = amount * taxPercentage / 100
        return taxValue

    @staticmethod
    def computeTotal(amount, taxPercentage):
        totalValue = amount + Tax.computeIva(amount, taxPercentage)
        return totalValue